.gitignore

```diff
@@ -1,2 +1,3 @@
 export/**
 !**/.gitkeep
+.idea
```

Comment:
Are you sure `.idea` is supposed to be in the `.gitignore`-file as opposed to a contributor's individual `.git/info/exclude`?